plugins/node/opentelemetry-instrumentation-ioredis/src/types.ts

```diff
@@ -39,11 +40,32 @@ export type DbStatementSerializer = (
   cmdArgs: IORedisCommand['args']
 ) => string;
 
+/**
+ * Function that can be used to add custom attributes to span on response from redis server
+ * @param span - The span created for the redis command, on which attributes can be set
+ * @param cmdName - The name of the command (eg. set, get, mset)
+ * @param cmdArgs - Array of arguments passed to the command
+ * @param response - The response object which is returned to the user who called this command.
+ *  Can be used to set custom attributes on the span
+ */
+export interface RedisResponseCustomAttributeFunction {
+  (
+    span: Span,
+    cmdName: IORedisCommand['name'],
+    cmdArgs: IORedisCommand['args'],
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    response: any
```

Comment:
Is there a better type available then `any`?
Maybe we should we use `unknown` instead to "force" the user to do some type check/cast.

Reply:
If it turns out that `unknown` is problematic/cumbersome to use we can still change it to `any`.
`any` to `unknonw` would be breaking for users.

Reply:
Cool so `unknown` is better in that case 👍 
Learned something new :)
Thanks for the review